specs/engage-digital_openapi3.yaml

```
@@ -5146,6 +5146,184 @@ paths:
       summary: Getting all timezones
       tags:
         - Timezones
+  /user_capacities:
+    get:
+      description: >-
+        This method renders all user capacities ordered by creation date (ascending).
+
+        Authorization​: only users who can manage topology.
+      operationId: getAllUserCapacities
+      parameters:
+        - description: The record index to start. Default value is 0.
+          in: query
+          name: offset
+          required: false
+          schema:
+            format: int32
+            type: integer
+        - description: >-
+            The max number of records to return. Default value is 30, max value
+            is 150.
+          in: query
+          name: limit
+          required: false
+          schema:
+            format: int32
+            type: integer
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+               $ref: '#/components/schemas/GetAllUserCapacities'
+          description: Success
+      summary: Getting all User Capacities
+      tags:
+        - UserCapacity
+    post:
+        description: >-
+          This method creates a new user capacity. In case of success it renders the
+          updated record, otherwise, it renders an error (422 HTTP code).
+          Please note that the order of the parameters is important, as the default and
+          max capacities will apply to the immediately above channel_id.
+
+          Authorization​: only users who can manage topology.
+        operationId: createUserCapacity
+        parameters:
+          - in: path
+            name: userCapacityId
+            required: true
+            schema:
+              type: string
+          - description: User Capacity name.
+            in: query
+            name: name
+            required: false
+            schema:
+              type: string
+          - description: The channel id on which apply the capacity.

```

Not sure about this one, there's probably a better way to say this
```suggestion
          - description: The id of the channel on which the capacities applies.
```

I'll let you have a look at the other occurrences